Source/tvOS/ASImageNode+tvOS.m

```diff
@@ -169,7 +169,7 @@ - (void)setFocusedState
   layer.shadowColor = [UIColor blackColor].CGColor;
   layer.shadowRadius = 12.0;
   layer.shadowOpacity = 0.45;
-  layer.shadowPath = [UIBezierPath bezierPathWithRect:self.layer.bounds].CGPath;
+  layer.shadowPath = CGPathCreateWithRect(self.layer.bounds, NULL);
```

Comment:
I think these will leak, since `CALayer.shadowPath` copies the argument. Extra semi-colon.

Reply:
Yep! will fix.